packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -486,6 +511,100 @@ describe('StripeWebhookHandler', () => {
       });
     });
 
+    describe('handleProductUpdatedEvent', () => {
+      let scopeContextSpy, scopeSpy, captureMessageSpy;
+      beforeEach(() => {
+        captureMessageSpy = sinon.fake();
+        scopeContextSpy = sinon.fake();
+        scopeSpy = {
+          setContext: scopeContextSpy,
+        };
+        sandbox.replace(Sentry, 'withScope', (fn) => fn(scopeSpy));
+        sandbox.replace(Sentry, 'captureMessage', captureMessageSpy);
+      });
+
+      it('throws a sentry error if the update event data is invalid', async () => {
+        const updatedEvent = deepCopy(eventProductUpdated);
+        updatedEvent.data.object.metadata['product:termsOfServiceDownloadURL'] =
+          'https://FAIL.cdn.mozilla.net/legal/mozilla_vpn_tos';
+        await StripeWebhookHandlerInstance.handleProductUpdatedEvent(
+          {},
+          updatedEvent
+        );
+
+        assert.called(
+          StripeWebhookHandlerInstance.stripeHelper.fetchPlansByProductId
+        );
+        assert.isTrue(scopeContextSpy.called, 'Expected to call Sentry');
+        assert.isTrue(captureMessageSpy.called, 'Expected to call Sentry');
+        assert.equal(validPlanList.length, captureMessageSpy.callCount);
+      });
+
+      it('does not throw a sentry error if the update event data is valid', async () => {
+        const updatedEvent = deepCopy(eventProductUpdated);
+        await StripeWebhookHandlerInstance.handleProductUpdatedEvent(
+          {},
+          updatedEvent
+        );
+
+        assert.isTrue(scopeContextSpy.notCalled, 'Expected not to call Sentry');
+      });
+    });
+
+    describe('handlePlanUpdatedEvent', () => {
+      let scopeContextSpy, scopeSpy;
+      beforeEach(() => {
+        scopeContextSpy = sinon.fake();
+        scopeSpy = {
+          setContext: scopeContextSpy,
+        };
+        sandbox.replace(Sentry, 'withScope', (fn) => fn(scopeSpy));
```

Comment:
The patch verifies that `captureMessage` was called in some of the tests above.  We should do that for the plan updated handler tests also.